### ISSUE TITLE
Component pages - quick panel

### ISSUE THREAD

**Issue by troussej** [jc7447/BetterDynAdmin#80]
Bookmared properties/methods for current component could be shown in a quick panel

**Comment by jc7447** [comment]
I'm not sure to understand, bookmarked properties and methods are already displayed for each favorite right ?

**Comment by troussej** [comment]
Yeah. The goal here is to have a quick access panel, always at the same place to quickly access these options on the current component, instead of having to search that in the bookmark bar.

Of couse we should consider the display/cluttering issues that it might cause.

**Comment by troussej** [comment]
also we could have quick access for current component event if it's not in the bookmarks (logDebug, different sections of the page, and default methods/properties)
